lib/events/complete.go

```diff
@@ -78,6 +80,16 @@ func (cfg *UploadCompleterConfig) CheckAndSetDefaults() error {
 	return nil
 }
 
+var (
+	incompleteSessionUploads = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: teleport.MetricIncompleteSessionUploads,
+			Help: "Number of sessions not yet uploaded to auth",
+		},
+	)
+	incompletePC = []prometheus.Collector{incompleteSessionUploads}
```

Comment:
I would remove this, and just `RegisterPrometheusCollectors(incompleteSessionUploads)` below.